README.md

```diff
@@ -1,30 +1,51 @@
-<h1 align="center">A official AsyncAPI React Component</h1>
+# AsyncAPI React Component
 
 ## Overview
 
-A official [React](https://reactjs.org/) component for AsyncAPI specification. It allows to validation and render AsyncAPI specification and use own styles.
+This project is an official [React](https://reactjs.org/) component for AsyncAPI specification. It allows you to validate and render AsyncAPI specification using your own styles.
 
-## Philosophy
+### Project structure
 
+This repository has the following structure:
 
+```
+  ├── docs                        # Directory with project-related documents
+  ├── library                     # Structure of the AsyncApi React component
+  │    ├── src                    # Source code of the AsyncApi React component
+  │    │    ├── components        # Source code of generic components used in the "containers" directory
+  │    │    ├── config            # Configuration of the AsyncApi React component
+  │    │    ├── containers        # Subcomponents for specific parts of the AsyncApi React component
+  │    │    ├── helpers           # Various helper functions
+  │    │    └── theme             # Theme-related files for the AsyncApi React component
+  │    └── test                   # Tests for the AsyncApi React component
+  └── playground                  # Structure of the Playground application for the AsyncApi React component
+       ├── public                 # Fonts, images, and icons used in the Playground application
+       ├── src                    # Source code of the Playground application
+       │    ├── common            # Various helper functions including mocks
+       │    └── components        # Components used in the Playground application
+       └── test                   # Tests for the Playground application
+```
+
+## Prerequisites
+
+Use the following tools to develop the AsyncApi React component:
 
-## Live Demo
+* React (version 16.0.0 or higher)
+* [Styled-components](https://github.com/styled-components/styled-components) (version 4.0.0 or higher)
+* TypeScript (version 3.0.0 or higher)
 
-A [live playground](https://magicmatatjahu.github.io/asyncapi-react/) is hosted on gh-pages.
+## Installation
 
-## Install
+Run this command to install the component:
 
 ``` sh
 $ npm install --save asyncapi-react
 ```
 
-## Requirements
-* TypeScript ^3.0.0
-* React ^16.0.0
-* [`styled-components`](https://github.com/styled-components/styled-components) ^4.0.0
-
 ## Usage
 
+Import the component to you project under the desired location:
```

Comment:
This sentence is strange in this context
if you want to introduce the code snippet somehow, just write
```
Add this code to your application to use the component. Make sure to pass stringified AsyncAPI specification into the `schema` property
```

Reply:
To `AsyncApi` component you can pass string or AsyncApiInterface provided by lib and this is also described in Params section, so for my opinion your describing is unnecessary.

Reply:
this is description of example, not of all props in details

Reply:
for now example is passing some artificial mock